src/Areas/Areas.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import { aPullAreas } from '../util/ApiCalls';
+import Card from '../Card/Card';
+
+class Areas extends Component {
```

Comment:
Nice choice to make this a class component. It's main function is to display info on the DOM, but it is much cleaner for it to handle its own methods and be able to use lifecycle methods.

Reply:
The lifecycle methods are why I went with a class component. I was originally going to make it functional, but thought this might make things cleaner to do the fetch when the component mounts.